exporter/datadogexporter/go.mod

```diff
@@ -3,19 +3,19 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datado
 go 1.19
 
 require (
-	github.com/DataDog/agent-payload/v5 v5.0.89
+	github.com/DataDog/agent-payload/v5 v5.0.91
 	github.com/DataDog/datadog-agent/pkg/proto v0.47.0-rc.3.0.20230720220350-143f7b52181b
-	github.com/DataDog/datadog-agent/pkg/trace v0.47.0-rc.3.0.20230720220350-143f7b52181b
-	github.com/DataDog/datadog-api-client-go/v2 v2.14.0
+	github.com/DataDog/datadog-agent/pkg/trace v0.47.0-rc.3
```

Comment:
This is the issue - it downgrades to an older version that isn't compatible with other dependencies.

I'm looking into ways to fix it.

Reply:
Interesting why it's happening. We can bring back your PR if that one is passing